### DISCUSSION TITLE
4/18 - 4/20 Humanitarian Mapathon

### DISCUSSION THREAD

**Discussion by albertkun** [albertkun/23S-ASIAAM-191A#9] [Extra Credit]
### 5 points

# Prompt
>    ## What do you think are drawbacks to humanitarian mapathon events (if any)? Could you leverage them for a cause you are interested in?

After participating in the [International Humanitarian Mapathon](https://mapathon.la/en/), reflect on the above prompt with at least 2-3 sentences.

**Comment by paige-lee** [comment]
I participated in the Humanitarian Mapathon today during my Digital Humanities 131 class. We worked on manually labeling buildings in Bata, Equatorial Guinea using satellite images. One drawback we discussed during class is that GIS technology is still not yet advanced enough to identify the roofs of buildings from images in an automated fashion. Mapathon events encourage mass mapping efforts during a short window of time. However, outside of the mapathon’s time frame, remote areas like Bata, Equatorial Guinea may not normally receive much attention. In consequence, residents of under mapped areas may have difficulty navigating or keeping track of buildings in their neighborhoods. Thus, I am interested in improving access to and knowledge about remote, rural, off-the-grid areas.

**Reply by albertkun** [reply]
Thanks for sharing, the other side of the question is (you don't have to answer this though), how can we take the results of the mapping back to the residents, like giving them access to building data via paper maps? Smart phones? Just something to think about!

**Comment by alanamalika** [comment]
After participating in the Humanitarian Mapathon today, I was able to learn more about the role of GIS technology in social change as well as the technical skills required in mapping. However, it also made me realize how digital mapping is highly dependent on public interest. Even though the focus was to map remote areas, mapathons are still not truly applicable because only areas that are receiving humanitarian aid are available to map out. Moreover, the incentive to map as much as possible could mean that participants are rarely checking on the success and legitimacy of the campaigns that are attached to mapathon tasks. Despite so, I learned about ways to leverage events for things that I am interested in beyond mapping, since I also learned about an equivalent event called “Wiki-athon.” In the age of the internet I think it would be great to host events like a mapathon or a wiki-athon to ensure information on marginalized groups and remote areas are better covered.
